components/fonts/hack-ttf/Makefile

```diff
@@ -0,0 +1,64 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Tim Mooney <Timothy.V.Mooney@gmail.com>
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=	Hack
+COMPONENT_VERSION= 2.020
+COMPONENT_SUMMARY= Hack, a typeface designed for source code
+COMPONENT_PROJECT_URL= http://sourcefoundry.org/hack/
+COMPONENT_FMRI=		system/font/truetype/hack
+COMPONENT_CLASSIFICATION=	System/Fonts
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=      $(COMPONENT_NAME)-$(COMPONENT_VERSION).zip
+COMPONENT_ARCHIVE_URL=	\
+	https://github.com/chrissimpkins/Hack/releases/download/v$(COMPONENT_VERSION)/$(COMPONENT_NAME)-v2_020-ttf.zip
```

Comment:
Better use $(COMPONENT_ARCHIVE) here, modifying COMPONENT_ARCHIVE so that it fits?